v-next/hardhat/src/internal/cli/telemetry/sentry/subprocess.ts

```diff
@@ -24,7 +24,9 @@ if (process.argv.length !== 7) {
   process.exit(1);
 }
 
+log("Config path:", configPath);
 log("Received envelope to be sent to Sentry from a subprocess");
+log(JSON.stringify(JSON.parse(serializedEnvelope), null, 2));
```

Comment:
I have kept this in - arguably it is quite noisy for users if we ask them to enable debugging on their failing error. What do you think?